XwaForm.cs

```diff
@@ -1621,6 +1633,26 @@ void menuGoalSummary_Click(object sender, EventArgs e)
 			string output = "(global goals not included):\r\n----------\r\n" + generateGoalSummary();
 			new GoalSummaryDialog(output).Show();
 		}
+		void menuLibrary_Click(object sender, EventArgs e)
+		{
+			if (_fLibrary != null)
+				_fLibrary.Close();
+			_fLibrary = new FlightGroupLibraryForm(Settings.Platform.XWA, _mission.FlightGroups, flightGroupLibraryCallback);
+		}
+		void flightGroupLibraryCallback(object sender, EventArgs e)
+		{
+			foreach (FlightGroup fg in (object[])sender)
+			{
+				if (fg == null)
+					break;
+				newFG();
```

Comment:
Note for me, but newFG should be updated to return bool so it can break out here the same way TIE was done